apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -302,3 +303,8 @@ type IngressCapabilities struct {
 	IngressProvider      string `json:"ingressProvider,omitempty"`
 	CustomDefaultBackend bool   `json:"customDefaultBackend,omitempty"`
 }
+
+type RotateCertificateInput struct {
+	RotateCACertificates bool     `json:"rotateCACertificates,omitempty" norman:"default=false"`
+	RotateComponents     []string `json:"rotateComponents,omitempty"`
```

Comment:
we need to list available values - etcd, control, etc - using `norman:"type=array[enum],options=....` annotation